### ISSUE TITLE
Bug: MeetingBar keeps closing

### ISSUE THREAD

**Issue by sammachin** [leits/MeetingBar#230]
**App version**
3.2.1

**Describe the bug**
I keep noticing that meetingbar is not running in my toolbar, I'm not sure when or why it dies but I've probabbly had to relaunch it 2-3 times this morning.

**To Reproduce**
Sorry but I've no idea! it just dissapears, let me know if there's anywhere I can extract logs.

**Expected behavior**
Stay running in the toolbar

**Desktop (please complete the following information):**
MacOS 11.2.3 on M1 MacMini

**Comment by sammachin** [comment]
I've just found the crash report in Console, pasted the first section below, let me know if you want the whole thing, I wasn't certain it didn't contain private info so not posting it all here
Seems to happen on the hour when I've had a meeting in the previous hour
```
Process:               MeetingBar [8495]
Path:                  /Applications/MeetingBar.app/Contents/MacOS/MeetingBar
Identifier:            leits.MeetingBar
Version:               3.2.1 (116)
App Item ID:           1532419400
App External ID:       841070742
Code Type:             ARM-64 (Native)
Parent Process:        ??? [1]
Responsible:           MeetingBar [8495]
User ID:               501

Date/Time:             2021-04-06 15:00:43.765 +0100
OS Version:            macOS 11.2.3 (20D91)
Report Version:        12
Anonymous UUID:        C261EBEA-D18E-64B9-43DF-FA08D767A046


Time Awake Since Boot: 1100000 seconds

System Integrity Protection: enabled

Crashed Thread:        1  Dispatch queue: com.apple.root.user-interactive-qos

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Application Specific Information:
*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'NSWindow drag regions should only be invalidated on the Main Thread!'
terminating with uncaught exception of type NSException
abort() called
```

**Comment by leits** [comment]
Thanks for the report @sammachin!
I'll take a look at this today.

**Comment by sammachin** [comment]
I think I discovered the cause, I had `Send Notification to Join Next Meeting` set but I hadn't granted meetingbar notifications permission in macos settings. Now I've fiex that it seems stable.

I'd still say this is a bug that the app quits but at least its easier to work around now and reproduce

**Comment by leits** [comment]
It's great that you found it.
@jgoldhammer, could you take a look?

**Comment by jgoldhammer** [comment]
I cannot reproduce this bug locally. Maybe the reason is that I have no M1 - MacMini, but used a Intel macbook-pro-16...

When I disable the notifications for meetingbar in system settings...
![image](https://user-images.githubusercontent.com/3872101/115960864-0662cf80-a514-11eb-9997-4f45f4701ca1.png)

Meetingbar shows me:
![image](https://user-images.githubusercontent.com/3872101/115960842-ea5f2e00-a513-11eb-8d48-bf461309de87.png)

@sammachin, can you help out?
- Can you check if this message in meetingbar also appears for you?
- Can you post the settings of the notifications for meetingbar?

Thanks
Jens

**Comment by leits** [comment]
@sammachin, also please check the latest version.
It fixed a similar issue: #251
